lib/Doctrine/ORM/PersistentCollection.php

```diff
@@ -514,10 +514,10 @@ public function offsetGet($offset)
     public function offsetSet($offset, $value)
     {
         if ( ! isset($offset)) {
-            return $this->add($value);
+            $this->add($value);
```

Comment:
What I meant was this:

```php
if ( ! isset($offset)) {
    $this->add($value);
    return;
}
```

I totally agree that it should return any value since it's a `void` but we shouldn't change the behaviour ;)